doc/source/development/contributing_environment.rst

```diff
@@ -216,6 +216,8 @@ should already exist.
    python setup.py build_ext -j 4
    python -m pip install -e . --no-build-isolation --no-use-pep517
 
+Be sure to also install the `pre-commit hooks <https://pandas.pydata.org/pandas-docs/stable/development/contributing_codebase.html#pre-commit>`__.
```

Comment:
I think (not 100% sure) that the link should be of the form ``:ref:`per-commit hooks <contributing.pre-commit>` ``

Reply:
When I write :ref:`per-commit hooks <contributing.pre-commit>` , it doesn't show a hyperlink text in the Preview section.
![image](https://user-images.githubusercontent.com/63000127/198294796-203fe180-2533-487d-9f67-d0ed5c2e44e3.png)

Reply:
That's to be expected.  If you look at similar links in that document, e.g., ``:ref:`Continuous Integration <contributing.ci>` ``, you'll see the same thing.

What you should do is build the docs locally and then you can test that the link is correct.

Reply:
or you can download the artifact from the CI build https://github.com/pandas-dev/pandas/actions/runs/3337146888 (see "artifacts" at the bottom of the page)